
"""
    PyLucid own Exception's
"""


class IntegrityError(Exception):
    """
    Fehler bei einer DB-Transaktion
    """
    pass